docs/doc_templates/future.md

```diff
@@ -0,0 +1,36 @@
+---
+permalink: /docs/future
+title: The future of Keanu
+---
+
+The aim of this project is to develop a probabilistic programming environment that enables modellers to 
```

Comment:
We use American spellings, right? That would be a single `l` - i.e. modelers, modeling - throughout.